mpv-install.bat

```diff
@@ -238,11 +238,12 @@ exit 0
 	call :reg add "%key%\shell\open" /v "LegacyDisable" /f
 
 	:: Set open command
-	call :reg add "%key%\shell\open\command" /d "\"%mpv_path%\" %mpv_args% -- \"%%%%L" /f
+	call :reg add "%key%\shell\open\command" /d "\"%mpv_path%\" %mpv_args% -- \"%%%%L\"" /f
 
 	:: Add "play" verb
 	call :reg add "%key%\shell\play" /d "&Play" /f
-	call :reg add "%key%\shell\play\command" /d "\"%mpv_path%\" %mpv_args% -- \"%%%%L" /f
+	call :reg add "%key%\shell\play" /v "MUIVerb" /t REG_EXPAND_SZ /d "@%%SystemRoot%%\system32\unregmp2.exe,-9991" /f
```

Comment:
I don't think I want to use this method. Relying on a hard-coded resource number in another program seems too fragile to me, even if the resource numbers are stable in practice. Also, Windows Media Player (and hence unregmp2.exe) is an optional Windows component, so it's not guaranteed to be present in System32.

mpv-install.bat is supposed to be a simple script, and as far as I know, adding translations the proper way would require a binary DLL, so that's out-of-scope. Batch _is_ pretty limiting, and there a few other things that can't be done without an EXE or DLL, like opening multiple files as a playlist from Explorer. I'd be happy to see mpv-install.bat get replaced by a more fully-featured installer with better shell integration at some point, though that would have to be a different project.